sdk/go/common/apitype/ai.go

```diff
@@ -16,14 +16,46 @@ package apitype
 
 import "encoding/json"
 
-// Requests
+// SummarizeUpdateRequest
 
 type CopilotSummarizeUpdateRequest struct {
 	Query           string                 `json:"query"`
 	State           CopilotState           `json:"state"`
-	DirectSkillCall CopilotDirectSkillCall `json:"directSkillCall"`
+	DirectSkillCall CopilotSummarizeUpdate `json:"directSkillCall"`
 }
 
+type CopilotSummarizeUpdate struct {
+	Skill  string                       `json:"skill"` // The skill to call. e.g. "summarizeUpdate"
```

Comment:
Given the type is called `CopilotSummaryzeUpdate, wouldn't the skill always be `summarizeUpdate`?